languages/csharp/reference/README.md

```diff
@@ -4,6 +4,12 @@
 
 The C# concept exercises are based on concepts. The list below contains the concepts that have been identified for the C# language.
 
+### Programming Paradigm
```

Comment:
I'm not entirely sure this adds much to the Concepts document to be honest. This is more like something for the C# track's language description. Could you perhaps remove this from the PR?

Reply:
Hi @ErikSchierboom,

Happy to remove this from PR if you see this not fitting here.

Reply:
@senal If you could do that, great!